kiwi_post_run

```diff
@@ -98,18 +91,23 @@ echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
 echo "metadata $METADATA"
+echo "tagfile $TAGFILE"
 
 # Generate metada JSON file for container-feeder
 cat << EOF > $METADATA
 {
   "image": {
     "name": "$CONTAINER_RAW",
-    "tags": [ "$CONTAINER_TAG", "latest" ],
+    "tags": [ "${VERSION}-${RELEASE}", "${CONTAINER_TAG}-${RELEASE}", "$CONTAINER_TAG", "latest" ],
```

Comment:
Why the VERSION-RELEASE ? I believe VERSION is the package version. I think with the CONTAINER_TAG-RELEASE should be enough, right?

Reply:
Moreover, at some point we should stop using this technique of wrapping images with RPMs, and I would prefer not to have any code which has some relation with that.